### ISSUE TITLE
slow speed with command line in .bat file

### ISSUE THREAD

**Issue by nodeus** [jarikomppa/img2spec#5]
Very slow perfomance in cycle like this, if you want to process thousands of images...

`for /r png %%i in (*.png) do (
img2spectrum.exe %%i conv_8.isw -p ok/s%%~ni.png
)`

**Comment by jarikomppa** [comment]
Yes, I imagine it would be. That's not exactly the use case I thought of =)

**Comment by nodeus** [comment]
I agree, but I'd really like the reactivity from the application. I have a very interesting case to use! :)
Thanks a lot for img2spec!

**Comment by jarikomppa** [comment]
There's a couple ways to fix this, either by adding a batch mode into the application that would eat a list of files to convert. Another possibility would be to figure out what's taking so long to start the application in the conversion mode and to optimize it.. I can't remember if it opens a window, for example, which shouldn't be necessary for only doing the conversion.
I don't have the time right now to look into this, though.
